src/modules/counter-of-cows.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const countCows = (userInput, number) => {
+  const numberStr = String(number);
+  const inputStr = String(userInput);
+  const count = {
+    bulls: 0,
+    cows: 0,
+  };
+
+  for (let i = 0; i < 4; i++) {
```

Comment:
why hardcoding `4` - what if you'd like to play with 6 or 9-digit number?

also you'd want to evaluate whether users' input matches the number.length
and probably, to only contain numbers - i'd use regexp for this.

```suggestion
  for (let i = 0; i < numberStr; i++) {
```